tests/fixtures/edid_data.py

```diff
@@ -52,7 +52,7 @@ def edid_loader(request):
 
 
 @pytest.fixture(scope='function', params=(SCREEN_MODELS_WITH_MD5))
-def edid(request, edid_loader):
+def edid(request, edid_loader, fs):
```

Comment:
fs here?

Reply:
I thought that I needed to make things work but it turns out that I don't 😂